modules/release-notes/pages/index.adoc

```diff
@@ -10,13 +10,13 @@ TigerGraph Server {page-component-version}.0 was released on Aug 31st, 2022.
 The following is a list of new features and improvements:
 
 === Elasticity
-
+* Added the ability to xref:ha:set-up-elastic-cluster.adoc[launch on-demand elastic clusters for on-prem instances (Preview)].
 
 === Manageability
 
 
 === Security
-* Enhanced the xref:user-access:access-control-model.adoc#_role_based_access_control[role-based access control] system to allow data CRUD privileges to be granted at the type and attribute level.
+* Enhanced the xref:user-access:access-control-model.adoc#_role_based_access_control[role-based access control] system to allow data CRUD privileges to be granted at the type and attribute level (Preview).
 
 === Ecosystem integration
 * Added the ability to xref:data-loading:data-streaming-connector/azure-blob.adoc[stream data from Azure Blob Storage] to xref:data-loading:data-streaming-connector/index.adoc[Data Streaming Connector].
```

Comment:
to a Data Streaming Connector

Reply:
Here I am talking about Data Streaming Connector as a feature